sanctum-lst-list.toml

```
@@ -592,7 +592,7 @@ symbol = "kumaSOL"
 mint = "KUMAgSzADhUmwXwNiUbNHYnMBnd89u4t9obZThJ4dqg"
 decimals = 9
 token_program = "TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA"
-logo_uri = "https://arweave.net/JWHmaNgsEXlHE1qJquHm5OluAX3V9CBKITwVbWbwL6M"
+logo_uri = "https://arweave.net/V2W_NcNoLjfDRS_-_llDIom04X8BY35V5yOelv-ATkI"
```

```suggestion
logo_uri = "https://arweave.net/dBlyZZ8SRU8gbBLB_Ahil_NXCtsh67PI10mTq9qcMVA"
```

this appears to be the metadata json uri, edited link is whats linked in the json file